devenv.yaml

```diff
@@ -0,0 +1,14 @@
+inputs:
```

Comment:
Can you enhance the README.md and add the usage of devenv?
There is a section about dev environments.